src/main/kotlin/racingcar/domain/race/Round.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar.domain.race
+
+class Round(val value: Int) {
+
+    init {
+        require(value in MIN_ROUND..MAX_ROUND) { ROUND_OUT_OF_RANGE_ERROR }
+    }
+
+    companion object {
+        private const val MIN_ROUND: Int = 1
+        private const val MAX_ROUND: Int = 10
```

Comment:
I think rename this field for improve readability: 

```suggestion
        private const val MIN_ROUND_LIMIT: Int = 1
        private const val MAX_ROUND_LIMIT: Int = 10
```